packages/tablev2/PrimaryLinkCell.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from "react-emotion";
+import { textWeight } from "../shared/styles/styleUtils";
+import { linkReset } from "../shared/styles/styleUtils/resets/linkReset";
+import { themeTextColorPrimary } from "../design-tokens/build/js/designTokens";
+
+export const PrimaryLinkCell = styled("div")`
```

Comment:
what's the rationale of forcing users of the table to use a `PrimaryLinkCell` in case they want to render a link. i think it would feel good if the table styled all links like this by default, without me having to wrap things in yet another component - because i think there's rarely a case where i want to maintain the default link styling within a table.

Reply:
Only links in the first column are meant to have a heavier font weight.

Maybe instead of using a component, we could style anchor tags that are children of the cells. Something like:
```
[role="gridcell"] a {
    ${linkReset};
}

[role="gridcell"]:first-child a {
    ${textWeight("medium")};
}
```

What do you think?

Reply:
i'd actually love that!